packages/dart_firebase_admin/test/credential_test.dart

```diff
@@ -66,5 +67,40 @@ void main() {
         Credential.fromServiceAccount(fs.file('service-account.json'));
       });
     });
+
+    group('fromApplicationDefaultCredentials', () {
+      test(
+          'completes if `GOOGLE_APPLICATION_CREDENTIALS` environment-variable is valid service account JSON',
+          () {
+        platform = FakePlatform(
```

Comment:
Tests that modify global state should have an addTearDown that reset the global state,

As mentioned before, I'd rather not have the global variable. Maybe use Zones?
But if we stick with this, we should remove add an addTearDown (and probably create a test util to both set the var and do the tear-down)

Reply:
Yeah, we're using Zones now